src/libPMacc/test/memory/HostBufferIntern/copyFrom.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2015 Erik Zenker
+ *
+ * This file is part of libPMacc.
+ *
+ * libPMacc is free software: you can redistribute it and/or modify
+ * it under the terms of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * libPMacc is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with libPMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// #includes in "test/memoryUT.cu"
+
+/**
+ * Checks if data is copied correctly from device to
+ * host.
+ */
+struct CopyFromTest {
+
+    template<typename T_DIM>
```

Comment:
Could you please write `T_Dim` to full fill the [coding guide lines](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Coding-Guide-Lines)